conf/application.conf

```diff
@@ -142,50 +144,28 @@ microservice {
       enabled = false
     }
   }
-}
 
+  services {
+    timeout = 5 seconds
 
-# Microservice specific config
+    delay-response = 2 seconds
 
-wiremock-port = 11111
-wiremock-port = ${?WIREMOCK_PORT}
+    protocol = http
 
-Test {
-  mongodb {
-    uri = "mongodb://localhost:27017/api-subscription-fields-test?rm.monitorRefreshMS=4000&rm.failover=default"
+    push-pull-notification {
+      uri = "http://localhost:6701"
+    }
   }
-}
 
-Stub {
-  mongodb {
-    uri = "mongodb://localhost:27017/api-subscription-fields-test?rm.monitorRefreshMS=4000&rm.failover=default"
-  }
 }
 
-Dev {
-  mongodb {
-    uri = "mongodb://localhost:27017/api-subscription-fields"
-  }
-
-  microservice {
-    metrics {
-      graphite {
-        host = graphite
-        port = 2003
-        prefix = play.${appName}.
-        enabled = false
-      }
-    }
-
-    services {
-      timeout = 5 seconds
+# Microservice specific config
 
-      delay-response = 2 seconds
+wiremock-port = 11111
+wiremock-port = ${?WIREMOCK_PORT}
 
-      protocol = http
-    }
+Dev {
+    mongodb {
+    uri = "mongodb://localhost:27017/api-subscription-fields-test?rm.monitorRefreshMS=4000&rm.failover=default"
```

Comment:
remove the excess confi

Reply:
I think this is the same is it used to be so is OK